lib/Params.ml

```diff
@@ -41,7 +41,7 @@ let parens_if parens (c : Conf.t) ?(disambiguate = false) k =
 let parens c ?disambiguate k = parens_if true c ?disambiguate k
 
 let wrap_exp (c : Conf.t) ?(disambiguate = false) ?(fits_breaks = true)
-    ~parens ~loc source k =
+    ?(offset_closing_paren = 0) ~parens ~loc source k =
```

Comment:
Could you add a bit of documentation, especially the fact that it's used in the `Closing_on_separate_line` case?

Reply:
done!